charts/t8s-cluster/ci/gpu-flavor-values.yaml

```diff
@@ -1,4 +1,4 @@
-workers:
+computePlanes:
```

Comment:
Why is this renamed?

Reply:
Because of https://gitlab.teuto.net/4teuto/ops/k8s/t8s-engine/-/merge_requests/663